roadway_objects/test/roadway_objects_empty_map_launch_test.py

```diff
@@ -0,0 +1,188 @@
+# Copyright 2023 Leidos
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pathlib import Path
+import time
+import unittest
+
+from ament_index_python import get_package_share_directory
+
+from carma_perception_msgs.msg import ExternalObjectList
+
+from launch import LaunchDescription
+from launch.actions import TimerAction
+
+from launch_ros.actions import Node
+
+from launch_testing import post_shutdown_test
+from launch_testing.actions import ReadyToTest
+from launch_testing.asserts import assertExitCodes
+
+from lifecycle_msgs.srv import ChangeState
+from lifecycle_msgs.msg import Transition
+
+import pytest
+
+import rclpy
+import rclpy.node
+from rclpy.context import Context
+from rclpy.executors import SingleThreadedExecutor
+
+
+class TestHarnessNode(rclpy.node.Node):
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__("test_harness", *args, **kwargs)
+
+        self.change_state_client = self.create_client(
+            ChangeState, "node_under_test/change_state"
```

Comment:
Just to check my understanding, are we setting up a client for `node_under_test/change_state` here so that TestHarnessNode will not create its publisher until node_under_test is active?

Reply:
The `change_state_client` is actually from an older version of the implementation and should be removed. State transitions are now handled by the `try_transition_node`, which spins up a temporary client to do the same thing. This avoids test creators from having to constantly add the transition client to their test harnesses.